bmc_dump_entry.cpp

```diff
@@ -14,6 +15,8 @@ namespace bmc
 
 void Entry::delete_()
 {
+    auto filename = file;
+    auto dumpId = id;
```

Comment:
yes, using temporary variables to store the values in case the entry is deleted. ok, will pass the parameters by value to createPELOnDumpDelete and will remove the temporary variables.